app/views/propertyrepresentation/manage/manageAgentProperties.scala.html

```diff
@@ -14,52 +14,54 @@
  * limitations under the License.
  *@
 
-@import actions.requests.BasicAuthenticatedRequest
-@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @import config.ApplicationConfig
 @import models.searchApi.OwnerAuthResult
 @import models.propertyrepresentation.AgentDetails
 @import utils.Formatters
-
 @this(govukTable: GovukTable, mainLayout: views.html.mainLayout)
 
-
 @(      ownerAuthResult: OwnerAuthResult,
         agentCode: Long,
-        agentDetails:Option[AgentDetails], backLink: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+        agentDetails:Option[AgentDetails],
+        listYears: Option[Seq[String]],
+        backLink: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
+@ratingList = @{
+    listYears.fold("Rating lists for this property are not available yet") { years =>
```

Comment:
Should this text be pulled from the messages file so that it can be translated?

Reply:
This is just a text for testing. I forgot to replace it with "". Sorry for that.

Reply:
What do you mean just for testing sorry, is listYears being None not a valid scenario?